external-crates/move/crates/move-analyzer/editors/code/package-lock.json

```diff
@@ -1,3954 +1,3936 @@
 {
-	"name": "move",
```

Comment:
Largely unrelated (did not re-generate this file after re-formatting `package.json` in https://github.com/MystenLabs/sui/pull/17863)